hr_course/models/hr_course.py

```diff
@@ -180,6 +183,38 @@ def cancel_course(self):
         for record in self:
             record.write(record._cancel_course_values())
 
+    @api.model
+    def send_email(self, partner):
```

Comment:
use a more specific name for the method, as hr_course inherits mail.thread it can shadow original method